src/v2/models/AppState.js

```diff
@@ -77,6 +78,19 @@ export default Model.extend({
     return this.get('idx').neededToProceed.find((remediation) => remediation.name === formName);
   },
 
+  hasMoreThanOneAuthenticatorOption(formName) {
```

Comment:
name-nit: There may be future cases where we'd like to get the total number of authenticator options. In the spirit of generic function signatures, how about renaming this and changing the return type to an array?

```suggestion
  getAuthenticatorOptions(formName) {
```

Reply:
We don't have the use case for this yet, so I would prefer to keep it as returning a boolean. Once we need this, it will be very easy to refactor this function to support both `hasMoreThanOneAuthenticatorOption` and `getAuthenticatorOptions `

Reply:
we can make this function more generic right away..

hasRequiredAuthenticators(formName, count) {
    .
    .
   return authenticatorOptions.length > count;
}

Reply:
ok, I got a couple of comments on this function, so I will refactor it. I prefer Jordan's proposal, which is to return the options objects and let the consumer of the function decide what to do with them (either look at the length or iterate through them). Will make the change